tests/e2e-playwright/specs/admin/teacher/course-creation.spec.ts

```diff
@@ -50,17 +49,25 @@ describe( 'Create Courses', () => {
 		// Click "Start with default layout" button.
 		await wizardModal.startWithDefaultLayoutButton.click();
 
+		const courseOutline = coursesPage.courseOutlineBlock;
+		await courseOutline.click();
+		await courseOutline.addModuleOrLessonButton.click();
+		await courseOutline.addModuleButton.click();
+
+		const moduleBlock = courseOutline.moduleBlock;
+		await moduleBlock.title.fill( 'Module 1' );
+		await moduleBlock.addLessonField.fill( 'Lesson 1 in Module 1' );
+		await page.pause();
```

Comment:
I think this pause is not expected

Reply:
Removed here: https://github.com/Automattic/sensei/pull/6289/commits/e015b6cf0450248e0b8a207c81e15d4ea0a8e42a